app/controllers/searches_controller.rb

```diff
@@ -16,8 +16,16 @@ class SearchesController < ApplicationController
   include QueryRoutableController
 
   def index
+    if @inactive_affiliate
+      @page_title = "Search Temporarily Unavailable - #{@inactive_affiliate.display_name}"
+      respond_to do |format|
+        format.html { render :inactive_affiliate, layout: 'application' }
+        format.json { render json: { error: "This search affiliate has been turned off" }, status: :service_unavailable }
+      end
+      return
```

Comment:
Do not use `return` to stop the execution flow please. I've pointed this out multiple times in the past.